### ISSUE TITLE
First skip link doesn't have focus indicator

### ISSUE THREAD

**Issue by aevanson** [PublicisSapient/accessible-ecommerce-demo#181]
skip navigation link isn’t getting focus indicator

Investigated this and it's due to the fact that the outline has an offset of 2px. This will need to be reduced for specific elements like this that have this issue.

**Comment by brianelton** [comment]
fixed on prod
